src/node_osrm_support.hpp

```diff
@@ -733,6 +733,74 @@ argumentsToTripParameter(const Nan::FunctionCallbackInfo<v8::Value> &args,
         return trip_parameters_ptr();
     }
 
+    if (obj->Has(Nan::New("roundtrip").ToLocalChecked()))
+    {
+        auto roundtrip = obj->Get(Nan::New("roundtrip").ToLocalChecked());
```

Comment:
I tried to change it to the Get pattern but can't figure it out right now. But I'm going to leave this pattern as is for now  as it is the pattern is used by the rest of the file. In a different PR, I can go through and convert all of them in one go.